packages/react-hooks/package.json

```diff
@@ -28,7 +28,6 @@
     "@fluentui/eslint-plugin": "^1.0.0-beta.0",
     "@types/enzyme": "3.10.3",
     "@types/enzyme-adapter-react-16": "1.0.3",
-    "@types/jest": "~24.9.0",
```

Comment:
I think removing this may cause errors from the `import/no-extraneous-dependencies` rule.

Reply:
with my proposal that lint rule should not be needed :)

Reply:
The lint rule would still be pretty important for preventing implicit deps in production code, but I see how it would become less important for tests.